src/test/scala/chiselTests/QueueSpec.scala

```diff
@@ -202,70 +202,70 @@ class QueueSpec extends ChiselPropSpec {
   implicit val noShrinkInt = Shrink[Int](_ => Stream.empty)
 
   property("Queue should have things pass through") {
-    forAll(vecSizes, safeUIntN(20), Gen.choose(0, 15)) { (depth, se, tap) =>
+    forAll(vecSizes, safeUIntN(20), Gen.choose(0, 15), Gen.choose(0, 1)) { (depth, se, tap, isSync) =>
```

Comment:
couldn't this just chose `true`, `false` and then that is passed directly instead of `isSync == 1`?

Reply:
Actually it doesn't work directly with `Gen.choose(true, false)` but it does with `Gen.oneOf(true, false)`. Thanks for the suggestion.